src/ethereum/crypto/blake2.py

```diff
@@ -151,7 +151,7 @@ def get_blake2_parameters(self, data: bytes) -> Tuple:
 
     def G(
         self, v: List, a: Uint, b: Uint, c: Uint, d: Uint, x: Uint, y: Uint
-    ) -> List:
+    ) -> Self:
```

Comment:
This change seems like it should have some corresponding changes elsewhere, because the return type is changing. Is that not the case?